ios_widget_catalogue_compare/ios/Runner/OverlayFlutterViewController.swift

```diff
@@ -0,0 +1,150 @@
+import SwiftUI
+
+
+/// The initial `UIViewController` defined in `Main.storyboard` uses this class.
+/// It is overlayed on top of the Flutter controls that are defined in `../lib`
+/// It will relay touch events to Flutter, so that the same touch event will reproduce
+/// with both Flutter and Swift.
+
+@available(iOS 14.0, *)
+class OverlayFlutterViewController: FlutterViewController, FlutterStreamHandler, ObservableObject {
+  
+  // Widget catalogue
+  @Published var controlKey: String = ""
+    
+  // SUI Overlay
+  lazy var overlaySUIView: OverlaySwiftUIView = setOverlaySUIView()
+  func setOverlaySUIView() -> OverlaySwiftUIView {
+    OverlaySwiftUIView(controller: self)
+  }
+  var originalOverlayCenter: CGPoint = .zero
+  
+  lazy var suiController: UIViewController = setSUIController()
+  func setSUIController() -> UIViewController {
+    UIHostingController(rootView: OverlaySwiftUIView(controller: self))
+  }
+
+  // UI Controls
+  var dropDownButton: UIButton = UIButton(frame: CGRect(x: 13, y: 50, width: 300, height: 40))
+
+  var xSlider: UISlider = UISlider()
+  var ySlider: UISlider = UISlider()
+  var alphaSlider: UISlider = UISlider()
+  
+  lazy var slidersStackView: UIStackView = setSlidersStackView()
+  func setSlidersStackView() -> UIStackView {
+    UIStackView(arrangedSubviews: [xSlider, ySlider, alphaSlider])
+  }
```

Comment:
As a bonus, it would be nice to have some sort of reset button for the offset as well